lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -147,8 +147,9 @@ def cmd_creds_help
     print_line "  -p,--port <portspec>  List creds with logins on services matching this port spec"
     print_line "  -s <svc names>        List creds matching comma-separated service names"
     print_line "  -u,--user <text>      List users that match this text"
-    print_line "  -t,--type <type>      List creds that match the following types: #{allowed_cred_types.join(',')}"
+    print_line "  -t,--type <type>      List creds of the specified type: password, ntlm, hash or any valid JtR format"
     print_line "  -O,--origins <IP>     List creds that match these origins"
+    print_line "     --realm            List creds that match this realm"
```

Comment:
Would it be possible to have this option be `-r` for the shorthand as we aren't using this at the moment for anything else and the only other similar option is `-R` for rhosts?